src/coreclr/src/inc/sstring.h

```diff
@@ -46,6 +46,10 @@
 #include "utilcode.h"
 #include "sbuffer.h"
 
+#ifndef _ASSERTE
+#define _ASSERTE(x)
```

Comment:
Would it be better to include debugmacros.h instead?

IIRC, we had cases where code like this disabled asserts in large parts of the code and allowed bugs to sneak through.

Reply:
I would love to add the right piece of code.  I didn't think `debugmacros.h` was working.  I eventually gave up as I saw a similar pattern in other files.

I'll try it again.

Reply:
Done